content/blog/how-to-provision-hpe-greenlake-private-cloud-resources-from-servicenow-service-catalog.md

```diff
@@ -0,0 +1,155 @@
+
+* From ServiceNow, go to **Morpheus Catalog** and select
+  **Properties**
+
+* Configure the following properties:
+
+  * The Morpheus Appliance endpoint: Enter the full URL to your HPE
+    GreenLake for private cloud appliance
+  * Username: Enter the name of the HPE GreenLake for private cloud
+    Service user(Received from HPE Support) that the Morpheus plugin
+    use to connect to the HPE GreenLake Private Cloud API
+  * Password: Enter the HPE GreenLake for private cloud Service user
+    password (Received from HPE Support)
+
+![](/img/figure1.png)
+
+## Integrating ServiceNow with HPE GreenLake for private cloud
+
+<!--StartFragment-->
+
+* From HPE GreenLake Central, locate the HPE GreenLake for private cloud service card and click the Launch icon to open the HPE GreenLake for private cloud dashboard
+* Navigate to Administration > Integrations
+* From the NEW INTEGRATION drop-down list, select ITSM ServiceNow
+* The NEW SERVICENOW INTEGRATION dialog box opens
+* From the dialog box, configure the ServiceNow integration parameters
+  * Name - Enter the integration name
+  * ENABLED - Select to enable consumption of this ServiceNow integration in HPE GreenLake for private cloud. The integration is enabled by default
+  * SERVICE NOW HOST-Enter the ServiceNow instance host URL (example: [https://your.instance.service-now.com](https://your.instance.service-now.com/)).
+  * USER - Enter a ServiceNow user that has below roles
+    * catalog_admin
+    * itil
+    * rest_service
+    * import_transformer
+    * xmodamorpheus_ca.integration
+  * PASSWORD -- Password of above-mentioned user
+  * Optional variables (CMDB CUSTOM MAPPING, CMDB CLASS MAPPING DEFAULT CMDB BUSINESS CLASS) not required for this use case
+* Click SAVE CHANGES
+
+The ServiceNow integration is now displayed in the list of integrations. Sample integration summary
+
+<!--EndFragment-->
+
+
+![](/img/figure2.png)
+
+## Expose private cloud resources to the ServiceNow Service Catalog
+
+After creating catalog items in the HPE GreenLake for private cloud as
+discussed in the previous
+[blog](https://developer.hpe.com/blog/curate-and-expose-service-catalog-items-using-hpe-greenlake-for-private-cloud/),
+the catalog items can be made available to ServiceNow service catalog by
+following procedure
+
+* Navigate to Administration > Integrations
+* Select the ServiceNow integration
+* From the Catalog Items tab, click **+ ADD CATALOG ITEM**
+* Select the catalog item using the drop-down and click **SAVE CHANGES**
+
+![](/img/figure3.png)
+
+Below is the sample list of catalog items exposed to the ServiceNow
+integration
+
+![](/img/figure4.png)
+
+Now, from ServiceNow, verify the availability of the resources as
+follows:
+
+* Log in to ServiceNow to access Service Catalog
+* From the Service Catalog, access the Morpheus plugin
+
+![](/img/figure5.png)
+
+* Exposed  HPE GreenLake private cloud resources are now available in
+  ServiceNow as Service Catalog
```

Comment:
you need an article prior to "Service Catalog". In this case, I think "a" might work, but it might be more appropriate to use "the".  Also, I would have made this a statement/plain text as opposed to a bullet and make sure to place a period at the end of the sentence.

Reply:
Converted it as a plain statement